### PR TITLE
update orbiters page

### PR DESCRIPTION
**Pull request by eshaben**
### Description

Updates the following items on the Orbiters page:

- the onboarding process list to apply to all networks
- adds moonbeam variables 
- switches program collators to orbiter pools
- adds author mapping bond info

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Goes with mkdocs PR: https://github.com/PureStake/moonbeam-mkdocs/pull/89

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [x] No additional PRs are required after the translations are done

#### Items to be Updated

n/a

### CODE REVIEW

File: node-operators/networks/collators/orbiter.md [hunk 1]
```diff
@@ -9,140 +9,161 @@ description: Learn about the Moonbeam Orbiter Program for collators including th
...
+- Orbiter’s transactions per block is within two standard deviation of the seven day program mean 
+- Orbiter’s block weight is within two standard deviation of the seven day program mean
 
 !!! note
     These factors are subject to change during this trial period.
 
 ## Leaving the Program {: #leaving-the-program }
 
-An Orbiter may leave the program and receive their bond back without any delay. The only limitation on this is if the Orbiter is currently active they cannot leave; once they are no longer active they can leave at any time by issuing an extrinsic.
+An orbiter may leave the program and receive their bond back without any delay. The only limitation on this is if the orbiter is currently active they cannot leave; once they are no longer active they can leave at any time by issuing an extrinsic.
```
Comment (artkaseman) [inline node-operators/networks/collators/orbiter.md:169]: let's add the extrinsic to use - we get these questions on the discord.  `moonbeamOrbiters.orbiterUnregister()`

File: node-operators/networks/collators/orbiter.md [hunk 2]
```diff
@@ -9,140 +9,161 @@ description: Learn about the Moonbeam Orbiter Program for collators including th
...
-- Orbiter’s block production is within 2 standard deviation of the 7 day program mean
-- Orbiter’s transactions per block is within 2 standard deviation of the 7 day program mean 
-- Orbiter’s block weight is within 2 standard deviation of the 7 day program mean
+- Orbiter has produced a block within the last three rounds they were active 
+- Orbiter’s block production is within two standard deviation of the seven day program mean
+- Orbiter’s transactions per block is within two standard deviation of the seven day program mean 
+- Orbiter’s block weight is within two standard deviation of the seven day program mean
 
 !!! note
     These factors are subject to change during this trial period.
```
Comment (artkaseman) [inline node-operators/networks/collators/orbiter.md:165]: let's get rid of trial period, something like 
These factors are subject to change as the program progresses

**Comment by eshaben** [comment]
@artkaseman updated with your feedback if you want to check it out again!

**Review by artkaseman — APPROVED** [review]
looks good

**Pull request merged**